.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
I'm not sure if we want to introduce `.editorconfig` just yet,
I pinged the issue here just now: https://github.com/hapijs/contrib/issues/38

Also we'd want to add this to it:
```
[*.{json,yml}]
indent_size = 2
```

Reply:
I'm sure there could be more things that could be added when reviewing the eslint hapi config, I haven't gone over it yet

Reply:
I can remove `.editorconfig` then

Reply:
👌 But I would be interested in reopening that discussion!

Reply:
Seems to be `done`